### ISSUE TITLE
"__v" warnings

### ISSUE THREAD

**Issue by henrik** [slime-lang/slime#41]
Warnings when booting Phoenix:

> web/views/layout_view.ex:2: warning: the underscored variable "__v" is used after being set. A leading underscore indicates that the value of the variable should be ignored. If this is intended please rename the variable to remove the underscore

This looks a likely culprit: https://github.com/doomspork/slim_fast/blob/c1b581fc0e222d54eeef98761f72c3fa874b28dd/lib/slim_fast/compiler.ex#L18

**Comment by doomspork** [comment]
You're on top of it @henrik, thanks!  Would you like to take a stab at fixing this one?  I haven't reproduced it myself but it suspect it will be straightforward.

**Comment by henrik** [comment]
Sure, I'll give it a shot!

**Comment by henrik** [comment]
I'm on Elixir 1.1.1 – maybe the warning is a new thing.

**Comment by doomspork** [comment]
Edit:  Nevermind.  I'll check out 1.1.1.

These all work as expected but none raise a warning:

``` elixir
# Controller
def false_attribute_value, do: false
def string_attribute_value, do: "test"
```

``` slim
/! Slim
.container role="main" test=false_attribute_value
.container role="main" test=string_attribute_value
```

``` html
<!-- HTML -->
<div class="container" role="main">
<div class="container" role="main" test="test">
```

**Comment by doomspork** [comment]
Definitely a 1.1.1 warning, just confirmed.  Have a go at it @henrik :+1:

**Comment by henrik** [comment]
How about this: https://github.com/doomspork/slim_fast/pull/42

**Comment by doomspork** [comment]
Closed via #42, thanks again @henrik :+1:
